src/app/collection/settings/tab/rules/add-rule-form/auto-link-form/auto-link-form.component.html

```diff
@@ -0,0 +1,39 @@
+<div [formGroup]="form">
+  <div class="form-group d-flex flex-wrap align-items-center justify-content-start">
+    <div class="text-nowrap" i18n="@@collection.config.tab.rules.autoLink.linkType">
+      Link record via
+    </div>
+    <select-item
+      class="ml-3"
+      emptyValue="Select a link type"
+      placeholderIcon="fas fa-link"
+      [items]="selectItems"
+      [selectedId]="linkTypeId"
+      (select)="onSelectLinkType($event)">
+    </select-item>
+  </div>
+  <div class="form-group d-flex align-items-center justify-content-start text-nowrap mt-2">
+    <div class="text-nowrap" i18n="@@collection.config.tab.rules.autoLink.where">
+      Where
+    </div>
+    <select-item
+      class="ml-3"
+      (select)="onSelectAttribute1($event)"
+      [emptyValue]="i18n({id: 'collection.config.tab.rules.autoLink.selectFrom', value: 'Select from \{\{collection\}\}'}, {'collection': collection.name})"
+      [placeholderIcon]="collection.icon"
+      [items]="attributes1"
+      [selectedId]="attribute1Id">
+    </select-item>
+    <div class="text-nowrap ml-3" i18n="@@collection.config.tab.rules.autoLink.equals">
+      equals
+    </div>
+    <select-item
+      class="ml-3"
+      (select)="onSelectAttribute2($event)"
+      [emptyValue]="linkedCollection ? i18n({id: 'collection.config.tab.rules.autoLink.selectFrom', value: 'Select from \{\{collection\}\}'}, {'collection': linkedCollection.name}) : i18n({id: 'collection.config.tab.rules.autoLink.linkTypeFirst', value: 'Select link type first'})"
```

Comment:
Every time change detection is performed the string is created. You can create these strings in component and then use it here.